src/coreclr/jit/lowerxarch.cpp

```diff
@@ -397,10 +397,12 @@ void Lowering::LowerBlockStore(GenTreeBlk* blkNode)
         {
         TOO_BIG_TO_UNROLL:
 #ifdef TARGET_AMD64
-            blkNode->gtBlkOpKind = GenTreeBlk::BlkOpKindHelper;
+            blkNode->gtBlkOpKind =
+                blkNode->IsZeroingGcPointersOnHeap() ? GenTreeBlk::BlkOpKindLoop : GenTreeBlk::BlkOpKindHelper;
 #else
             // TODO-X86-CQ: Investigate whether a helper call would be beneficial on x86
-            blkNode->gtBlkOpKind = GenTreeBlk::BlkOpKindRepInstr;
+            blkNode->gtBlkOpKind =
+                blkNode->IsZeroingGcPointersOnHeap() ? GenTreeBlk::BlkOpKindLoop : GenTreeBlk::BlkOpKindRepInstr;
```

Comment:
The "too big to unroll" case is expected to be rare, right?

Should `getUnrollThreshold` be artificially reduced under stress to make the loop case happen more frequently?

And/or should we go down the BlkOpKindLoop even for non-GC-pointer-on-heap zeroing under some stress percentage?

Basically, do we get enough test coverage as-is, or should we artificially adjust test coverage sometimes under stress?